src/elements/content-preview/ContentPreview.js

```diff
@@ -366,8 +364,10 @@ class ContentPreview extends React.PureComponent<Props, State> {
 
         if (hasFileIdChanged) {
             this.destroyPreview();
+            this.setState({ selectedVersion: undefined });
             this.fetchFile(currentFileId);
         } else if (this.shouldLoadPreview(prevState)) {
+            this.destroyPreview();
```

Comment:
This seems like a fairly risky change since it'll affect lots of cases. Are we confident about it? Does it cause any visual issues when navigating between files?

Reply:
At the very least, the majority of the use cases were file id changing and that has always called `destroyPreview`, so changing files works fine. It appears to work fine switching between versions as well